src/components/Book.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+const Book = () => {
+  const [book] = useState({ title: 'Sample Book', author: 'John Doe' });
+
+  const deleteBook = () => {
+    // Logic to delete the book
+  };
+
+  return (
+    <div>
+      <h3>{book.title}</h3>
+      <p>{book.author}</p>
+      <button onClick={deleteBook} type="button">Delete</button>
+    </div>
+  );
```

Comment:
- [ ]  You've done well, 👏 , but lets refactor this to meet the requirements as expected. 

- Kindly make this a `Books` component, which contains the following components of which you've created the already. Calling and strctureing them properly is all that is left. :


- `BookList`

-- Renders a list of books using the `Book` component

- `Book` component
-- Takes the book data as a prop and renders the title and the author. It should Have a button for removing books
-- The button wouldn't remove the books yet, this is just a placeholder for now

- `NewBookForm`

-- Renders a form with inputs for the title and the author of the book
Has a button to submit. The button wouldn't add the book yet, this is just a placeholder for now

- [ ] Since you have the other components, kindly structure them properly 👍 

### Guide:
![image](https://github.com/CVILLA09/Bookstore/assets/50990925/703c793c-219d-40bf-96c4-d908fc1001f0)